### DISCUSSION TITLE
Data acquisition for gbm_pd1 vignette

### DISCUSSION THREAD

**Discussion by ejc043** [netZoo/netZooR#258] [Q&A] [answered]
Hi all -- thank you for the wonderful package! 
I have been struggling to find the data used in the gbm_pd1 vignette: Gene regulatory network analysis in Glioblastoma 
I know it's available through GRAND but the matrices seem different from the matrices used in the vignette?

**Comment by mararie** [comment, marked as answer]
Hi! This is because the full networks on GRAND were processed for the vignette to make it feasible for the user to run the vignette online, as the original patient-specific networks are very large. You can download the "Edg" object from GRAND to obtain the full networks. These are stored in a format that contains both information on edges. The first column is an identifier, the second column contains the name of the TF and the third column the name of the target gene that form a connection. The following columns contain edge weights for each individual sample. This matrix was processed in different ways that are shortly explained in the vignette—averaged per condition for the ALPACA analysis, and Gene Targeting Score (sum of edge weights that connect each individual TF to a specific gene) for the differential targeting analysis. Feel free to drop me a message here https://www.kuijjerlab.org/contact/ if this is not clear and/or you would like to discuss.
